### ISSUE TITLE
Concurrency issue

### ISSUE THREAD

**Issue by igashev** [parley-messaging/ios-library#14]
https://github.com/parley-messaging/ios-library/blob/744dcf7b1f354ab975bfd0364be39d85fafcbd41/Source/Views/ParleyView.swift#L147

This function seems to be called multiple times during a short period of time and causing an incorrect visual outcome.

I have tried different ways to fix it and there are several possible solutions.

One way is to get rid of the animation and leave it like the following:
```swift
private func syncStackView(_ changes: @escaping () -> Void) {
    changes()
    self.stackView.isHidden = !self.stackView.arrangedSubviews.contains { view -> Bool in return view.isHidden == false }
    self.syncMessageTableViewContentInsets()
}
```

Another way is to keep the animation and move the `completion` code inside the `animation` block.
```swift
private func syncStackView(_ changes: @escaping () -> Void) {
    UIView.animate(withDuration: 0, animations: {
        changes()
        self.stackView.isHidden = !self.stackView.arrangedSubviews.contains { view -> Bool in return view.isHidden == false }
        self.syncMessageTableViewContentInsets()
    })
}
```

This would result in a correct order of showing the views which are inside this `stackView`.

**Comment by alexkok** [comment]
Thanks @igashev ! We don't seem to be able to reproduce the incorrect visual outcome described here. Although the `animate` implementation here is also used to solve some visual issues faced earlier.

If the `animate` isn't needed anymore, we will remove it and go with the first suggestion.

We will test it out and make a new version that contains this fix.

**Comment by igashev** [comment]
When an asynchronous function is called multiple times in a short period of time, it is never guaranteed which call will finish first or last. Our case was that `stickyMessage` was set and the SDK took this into account but then when this function was called several times it just messed things up and ended up without showing the sticky message on top.

**Comment by alexkok** [comment]
Fixed in #15 and released in version 3.3.1 🚀

**Comment by igashev** [comment]
@alexkok Thanks! :)
